cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/goals/AbstractGoal.java

```diff
@@ -70,6 +70,11 @@ public void configure(Map<String, ?> configs) {
     _minMonitoredPartitionPercentage = parsedConfig.getDouble(MonitorConfig.MIN_VALID_PARTITION_RATIO_CONFIG);
   }
 
+  private static boolean hasNoExcludedBrokersForReplicaMoveWithReplicas(ClusterModel clusterModel, OptimizationOptions optimizationOptions) {
```

Comment:
Nit: My personal preference is to make this method `hasExcludedBrokersForReplicaMoveWithReplicas` instead of `hasNoExcludedBrokersForReplicaMoveWithReplicas`. I think that removing this negation in the method name (and meaning) improves the readability a little.